src/app/rosetta/network.ml

```diff
@@ -0,0 +1,88 @@
+open Core_kernel
+open Async
+open Models
+
+module Get_status =
+[%graphql
+{|
+  query {
+    bestChain {
+      stateHash
+      protocolState {
+        blockchainState {
+          utcDate @bsDecoder(fn: "Int64.of_string")
+        }
+        consensusState {
+          blockHeight @bsDecoder(fn: "Graphql.Decoders.int64")
+        }
+      }
+    }
+    daemonStatus {
+      peers
+    }
+  }
+|}]
+
+module Get_version = [%graphql {|
+  query {
+    version
+  }
+|}]
+
+(* TODO: Is there a better way to get the genesis block info *)
+let genesis_block_query =
+  Caqti_request.find Caqti_type.unit Caqti_type.string
+    "SELECT state_hash FROM blocks WHERE height = 0"
+
+let router ~graphql_uri (module Db : Caqti_async.CONNECTION) route body =
+  let open Async.Deferred.Result.Let_syntax in
+  match route with
+  | ["list"] ->
+      let%map _meta = Mappers.parse @@ Metadata_request.of_yojson body in
+      Network_list_response.to_yojson
+        { Network_list_response.network_identifiers=
+            [ { Network_identifier.blockchain= "coda"
+              ; network= "testnet"
+              ; sub_network_identifier= None } ] }
```

Comment:
I recall us wanting to use sub_network_identifier for something, but I don't remember now. Do you know? Anyway, we shouldn't block the PR on that anyway.